en/messages.json

```diff
@@ -296,5 +296,17 @@
   "learnWhatToDo": {
     "message": "Learn what to do",
     "description": "Link in banner alert"
+  },
+  "telemetry_screen_header":{
+    "message": "Help us improve the extension",
+    "description": "Header for the Telemetry Opt-In Screen"
+  },
+  "telemetry_toggle_text":{
+    "message": "Share technical data",
+    "description": "Label for the Telemetry Opt-In toggle"
+  },
+  "telemetry_screen_descr":{
+    "message": "Help us improve features and performance by sharing crash reports, diagnostics, and product use data. We never collect data about the sites you vist. Learn more in the $1.",
```

Comment:
"product use" or "product usage"?

`$1` is not the placeholder format commonly used in WebExtension, the string also needs a comment. See as an example another string in the file

```
  "websiteLocationLabel": {
    "message": "$SERVERLOCATION$: Mozilla VPN website location",
    "description": "Aria label for a flag icon representing the location of the VPN server",
    "placeholders": {
      "serverlocation": {
        "content": "$1",
        "example": "Germany"
      }
    }
  },
  ```